lib/cisco_node_utils/cmd_ref/bridge_domain.yaml

```diff
@@ -3,6 +3,10 @@
 ---
 _exclude: [N3k, N5k, N6k, N8k, N9k, ios_xr]
 
+_template:
+  get_command: "show running-config bridge-domain"
+  get_value: '/^bridge-domain[\s\S]*bridge-domain <bd>\s+(shutdown)?\s*(fabric-control)?\s*(?:name (\S+)|\s*)/'
```

Comment:
See my earlier comment about misusing your template (https://github.com/cisco/cisco-network-node-utils/pull/302/files#diff-dcbfe5a528545ac12dacc955d843bae3R148)

Use something like this instead:

```
_template:
  get_command: "show running-config bridge-domain"
-  get_value: '/^bridge-domain[\s\S]*bridge-domain <bd>\s+(shutdown)?\s*(fabric-control)?\s*(?:name (\S+)|\s*)/'
+ context: '^bridge-domain <bd>'
```